app/code/Magento/CatalogImportExport/Test/Unit/Model/Import/Product/ValidatorTest.php

```diff
@@ -64,7 +65,10 @@ protected function setUp(): void
         $this->objectManagerHelper = new ObjectManagerHelper($this);
         $this->validator = $this->objectManagerHelper->getObject(
             Validator::class,
-            ['validators' => $this->validators]
+            [
```

Comment:
Can we create this validator with `$this->validator = new Validator(...params);` ?

Reply:
Hello @ihor-sviziev !
Thanks for your suggestions! I think in this case better to use `\Magento\Framework\TestFramework\Unit\Helper\ObjectManager` because we have some Mock dependencies. `($this->validators)` What do you think?